### PR TITLE
JBPM-5484 & JBPM-5550: Fixed icons and shapes for Task & Reusable sub-process

### PR DESCRIPTION
**Pull request by romartin**
Hey @hasys @manstis 
This commit fixes the issues for task & sub-process icons/shapes.
- [JBPM-5484](https://issues.jboss.org/browse/JBPM-5484) - AFAIK we should keep the none task support, so  this commit replaces the none task icon and makes it different from the user task. 
- [JBPM-5550](https://issues.jboss.org/browse/JBPM-5550) - Same for sub-processes, made different icons and fixed the reusable sub-process shape. 

Thanks!
Roger

### CODE REVIEW

**Comment by hasys** [comment]
Jenkins execute full downstream build

**Comment by romartin** [comment]
Hey @hasys 

Thanks for the review! The changes you requested are fine, but there was the need of some improvements in the code to achieve the SVG complex icons used on BPMN.

So once [this other PR](https://github.com/kiegroup/kie-wb-common/pull/948) gets merged, I'll move to this icons again and fix the issues.

Thanks!!

**Comment by romartin** [comment]
Hey @manstis  @hasys 

Just updated this commit with some updates and fixes for the @hasys  comments.

Some updates:
- Finally this ticket resolves [JBPM-5484](https://issues.jboss.org/browse/JBPM-5484), [JBPM-5550](https://issues.jboss.org/browse/JBPM-5550) and [JBPM-5919](https://issues.jboss.org/browse/JBPM-5919). 
- Fixed/improved some other BPMN icons as well.
- Improved support for the SvgDataUriGlyph type - now allows displaying "composite" SVG glyphs.

Fixed all above comments from @hasys  too:
- Fixed reusable subprocess shape displaying
- Fixed bug about resizing svg shapes - after saving and loading again the diagram, the shape inner icons were using a different size than the original one
- Fixed subprocesses icons - now are different from the gateway ones and are not partially diplayed/cut
- Changed icon for none task

@manstis As this commit fixes  a bug on the SVG shapes (about initial resizing), your SVG's for DMN could be affected, lemme know if both glyph/shapes are displaying good for you please!! Ping me if any issues!!

Thanks!

**Comment by hasys** [comment]
Jenkins execute full downstream build

**Comment by romartin** [comment]
hey @psiroky @hasys @manstis , that rest tests failing for the downstream build, are the same ones as last week, so same ones as from @manstis  bsig mail? You know who can fix this? This PR is taking long again due to this. (CC @krisv )

**Comment by psiroky** [comment]
@romartin if you take a look at the latest build of kie-wb-distributions (https://kie-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/kie-wb-distributions/) all the tests are green, which seems to suggest the REST tests were already fixed. Not sure why they are failing for this PR. Let's try another build.

Jenkins execute full downstream build

**Comment by psiroky** [comment]
Jenkins execute full downstream build

File: kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-client-common/src/main/java/org/kie/workbench/common/stunner/core/client/util/SvgDataUriGenerator.java [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+    }
+
+    private static SVGContentType getType(final String s) {
+        if (s.startsWith(SVG_DATA_URI_BASE64)) {
+            return SVGContentType.XML_BASE64;
+        } else if (s.startsWith(SVG_DATA_URI_UTF8)) {
+            return SVGContentType.XML_UTF8;
+        }
+        throw new IllegalArgumentException("The image data-uri specified is not a valid SVG data " +
+                                                   "for being emedded into the DOM.");
```
Comment (hasys) [inline kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-client-common/src/main/java/org/kie/workbench/common/stunner/core/client/util/SvgDataUriGenerator.java]: Minor typo: **emedded**

Comment (romartin) [inline kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-client-common/src/main/java/org/kie/workbench/common/stunner/core/client/util/SvgDataUriGenerator.java]: fixed locally, for next PR :) thanks!

**Comment by romartin** [comment]
@psiroky  great, thanks!! will look at master next time :)

@hasys yep agree, the plus icon for the emb. subprocess shape is not on center, I realized but couldn't spent more time on fixing it, will try to do so thanks!

@manstis  any issues in your SVG shapes/icons after this? I had to basically reduce the size of the svg files provided by UX (btw will let them know), as if those svg's are being used for shapes and icons, when scaling to really small sizes (icons case), some of the inner elements were so small that almost not visible. I did this because BPMN uses same svgs for shapes and icons now, will try to keep this approach and will see how it works, less  svg files to maintain for now...  Another option is using different SVG for shapes / icons too, that's up to the developer... Anyway thanks for review/merge too! Lemme know if any issues!

**Pull request merged**